### PR TITLE
LPS-202997 No Success message is shown when restoring KB Article with specific status

### PR DESCRIPTION
**Pull request by marcogalluzzi**
Solving this problem: https://liferay.atlassian.net/browse/LPS-202997

### CODE REVIEW

**Comment by marcogalluzzi** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/trash/KBArticleTrashHandler.java [hunk 1]
```diff
@@ -103,6 +106,17 @@ public TrashedModel getTrashedModel(long classPK) {
 			classPK, WorkflowConstants.STATUS_ANY);
 	}
 
+	@Override
+	public TrashRenderer getTrashRenderer(long classPK) throws PortalException {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/trash/KBArticleTrashHandler.java:110]: Question:

why there is no need of getTrashRenderer on `KBFolderTrashHandler`?

Comment (marcogalluzzi) [reply modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/internal/trash/KBArticleTrashHandler.java:110]: KBFolders assets are not rendered

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**